### ISSUE TITLE
No menu item to open saved local projects

### ISSUE THREAD

**Issue by ToonTalk** [ericrosenbaum/BeetleBlocks#225]
Without logging in I saved to the browser but I don't see a menu item for opening that saved project.

**Comment by bromagosa** [comment]
We removed that option, but you can always use the Control+O (Command+O on Mac) to bring up the project dialog :)

**Comment by ToonTalk** [comment]
Thanks. I'm very curious what the rationale was given that one can use the menu to save but not open saved projects.

**Comment by bromagosa** [comment]
Actually, the local option in the save menu shouldn't be there either. That's my mistake, let me rephrase and reopen this issue.

Thanks for noticing that!
